Directory.Packages.props

```diff
@@ -21,6 +21,7 @@
         <PackageVersion Include="Microsoft.ComponentDetection.Detectors" Version="$(ComponentDetectionPackageVersion)" />
         <PackageVersion Include="Microsoft.ComponentDetection.Orchestrator" Version="$(ComponentDetectionPackageVersion)" />
         <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
+        <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.0-rc.2.23479.6" />
```

Comment:
Does this need to be updated upstream? I'd like to avoid using an RC version if `8.0.0` is available.

Reply:
Yes, bumping to 8.0.0 in the meantime.